src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -384,12 +431,25 @@ export class OrganizationIntegrations extends AsyncComponent<
           <SettingsPageHeader
             title={title}
             action={
-              <SearchInput
-                value={this.state.searchInput || ''}
-                onChange={this.onSearchChange}
-                placeholder="Filter Integrations..."
-                width="25em"
-              />
+              <FlexContainer>
```

Comment:
Possible to give this more of a semantic name? Maybe `ActionContainer`?

Reply:
@EvanPurkhiser haha naming is one of the hardest things in programming. `ActionContainer` works for me.